docker/ravendb-ubuntu/run-raven.sh

```diff
@@ -10,6 +10,11 @@ if [ ! -z "$RAVEN_ARGS" ]; then
 	COMMAND="$COMMAND ${RAVEN_ARGS}"
 fi
 
-eval $COMMAND
-exit $?
+[ -n "$RAVEN_DATABASE" ] && export RAVEN_Setup_Mode=None
+$COMMAND &
+COMMANDPID=$!
+
+[ -n "$RAVEN_DATABASE" ] && ./create-database.sh
 
+wait $COMMANDPID
+exit $?
```

Comment:
`exit $?` is redundant. wait pid is going to exit with the exit code of the waited process

Reply:
done